.gitattributes

```diff
@@ -1,6 +1,5 @@
 * text=auto
 
-*.patch binary
```

Comment:
This should be kept.

Reply:
To be honest, I thought this was an artefact from times long past. Github already ignores it anyway, and I'd also like to be able to see the diff in my git locally before committing. What's the reason for keeping it?

Not that I care much about this, I can carry a local diff as well.

Reply:
This is not some left over junk. The python source code contains files that contain Windows line endings, so the patches to those files must also contain Windows line endings.

Reply:
@mingwandroid 
The patches applied fine without marking them as binary. It's a pain to keep the right line endings when editing files that use a different convention (regardless of whether they're marked "binary"), but it seems to me that the only difference is whether I can see a git diff locally or not.

Of course, I may be overlooking something else, and so I reverted that change.

Reply:
Yes you are. Git has the concept of "line ending check out mode", unfortunately. If we aren't explicit about the lines endings then people who check out the repo in lf mode well be unable to apply the patches. 

You did say you were new to git? I am far from new to it.

Reply:
> You did say you were new to git? I am far from new to it.

New to editing patch files by hand, yes; new to git, no.

Fair enough on the line endings. Guess my defaults took care of that.

In any case, I reverted it already. Any tips on the failures? Did you have a chance to review the patches I mentioned above?

Reply:
> New to editing patch files by hand, yes; new to git, no.

To work with theses patches,  I use a clone of the upstream git repo. I would never work on them by hand instead using git am,  rebase and format-patch.

They looked reasonable upon eyeballing and thanks for taking care there. I can't help or with the failures at present, sorry, I'm too busy.

Reply:
> To work with theses patches, I use a clone of the upstream git repo. I would never work on them by hand instead using git am, rebase and format-patch.

I was using git am and `git apply -v` to explore, and `format-patch` to generate new patches, but then pasting things into your patches so I don't change attribution. But yeah, I can see how using the upstream repo would help a lot there. :)

> I can't help or with the failures at present, sorry, I'm too busy.

OK, I'll try to have a look when I have more time.

Reply:
@h-vetinari, this file is autogenerated. The change will just be overwritten anyways. It would be good to revert it to it's previous autogenerated state. If you find issue with the contents of this file, that should be raised over at [conda-smithy]( https://github.com/conda-forge/conda-smithy ) not in individual feedstocks. Thanks for your help. 🙂